### ISSUE TITLE
Segoe Fluent Replacement Font

### ISSUE THREAD

**Issue by robloo** [amwx/FluentAvalonia#40]
I re-built the Segoe Fluent font using the Fluent UI System icons that are open source. I notice you are using Fluent UI System fonts and Unicode points as well with this project for the SymbolIcon.

Would you like to switch to use the replacement font created here?

https://github.com/robloo/SymbolIconManager/blob/main/Fonts/WinSymbols3.ttf
https://github.com/robloo/SymbolIconManager

Aside from being smaller in size and only 1 file, it would standardize all the Unicode points with WinUI upstream and align the glyphs for the most part with Fluent version 2 (the same ones you are using now single Segoe Fluent and Fluent UI System Icons are very similar).

The goal here is for every non-Microsoft XAML framework to use the same Segoe Fluent font replacement. Anyone can contribute mappings to share with others as well:

https://github.com/robloo/SymbolIconManager/blob/main/Source/Data/Mappings/SegoeFluent.json
https://github.com/robloo/SymbolIconManager/blob/main/Docs/IconMapping.md

**Comment by amwx** [comment]
Just to make sure I'm understanding you just remapped the Fluent UI icons to the same code points as Segoe Fluent so it's still Fluent UI icons? That's an interesting and great idea! Are both filled and regular icons combined into WinSymbols3, or at least capable of being added? Just making sure there's no loss of functionality if this switch is made.

But I do like the idea. I'm planning to do a pass over this library post Win 11 release to make sure everything is up to date with WinUI, adding some missing styles for a couple Avalonia controls & support for high contrast theme. So I'm open to swapping fonts in this time frame, one file & smaller size is always better :) . So long as any symbol referenced here, either in my Symbol enum or through a FontIcon in a template has a match, I don't see a problem.

**Comment by robloo** [comment]
> you just remapped the Fluent UI icons to the same code points as Segoe Fluent so it's still Fluent UI icons?

Yes, precisely. Fluent UI System icons have been remapped to the Segoe Fluent Unicode point, then I rebuilt the font using FontForge. The tooling for this is all automated. Additionally, I also take some icons from Line Awesome and WinJS Symbols (both MIT license) when Fluent UI System is missing a glyph. Line Awesome and WinJS Symbols are, however, more useful to rebuild the Segoe MDL2 assets font due to the glyph design language.

As can be seen from the mapping file below, the Bluetooth icon is coming from FluentUISystem-Regular with Unicode point F1DE (internally translated to the SVG named ic_fluent_bluetooth_20_regular.svg) and then passed to FontForge in the python build script.

```json
  {
    "Destination": {
      "IconSet": "SegoeFluent",
      "UnicodePoint": "E702",
      "Name": "Bluetooth"
    },
    "Source": {
      "IconSet": "FluentUISystemRegular",
      "UnicodePoint": "F1DE",
      "Name": "ic_fluent_bluetooth_20_regular"
    },
    "GlyphMatchQuality": "High",
    "MetaphorMatchQuality": "Exact",
    "IsPlaceholder": false,
    "Comments": ""
  },
```
> Are both filled and regular icons combined into WinSymbols3, or at least capable of being added?

Yes, both regular and filled variants are used already for source glyphs. I've done a full pass of the SegoeFluent icons font and mapped all glyphs I could find a match for (627 of 1404 are complete). 

> Just making sure there's no loss of functionality if this switch is made.

There should be no loss of functionality at all. WinSymbols3 is designed to be a drop-in replacement for the Segoe Fluent font shipping with Windows 11. If anything is missing its going to be really easy to add. The following source icon sets are already supported: FluentUISystemFilled, FluentUISystemRegular, LineAwesomeBrand, LineAwesomeRegular, LineAwesomeSolid. Just use them in a mapping. Custom glyphs (from SVG files) and using WinJS Symbols is also possible without too much work.

> So I'm open to swapping fonts in this time frame, one file & smaller size is always better :) 

Perfect! If you are missing anything just let me know and I'll add a mapping for you and rebuild the font (I should have got the vast majority already though).
